functions/compute-stats.js

```diff
@@ -69,10 +69,10 @@ exports.handle = function (event, context) {
   const categories = `https://stats.data.gouv.fr/index.php?module=API&format=json&idSite=145&period=day&method=Events.getCategory&module=API&showColumns=label&filter_limit=9999&date=${dateAsString}`;
 
   get(categories)
```

Comment:
You can actually entirely remove `functions/compute-stats.js` as it is no longer used